consumergroup/ddb.go

```diff
@@ -0,0 +1,169 @@
+package consumergroup
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+
+	consumer "github.com/harlow/kinesis-consumer"
```

Comment:
feels like we might be in danger of cyclical import. I've tried to stick to  general  rule of thump that  sub packages don't know about directory levels above them.

Reply:
I think we might be fine once we reorganize the packages, but if not, let's revisit this guy.  The only thing it is using right now is the Lease struct.

Reply:
I moved Lease to storage - in a new file called lease.go - all storage options should be able to use this.  Let me know if that works.